Remotion/Web/ClientScript/Web.ClientScript.csproj

```diff
@@ -10,5 +10,9 @@
     <AssemblyName>Remotion.Web.ClientScript</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
   </PropertyGroup>
+  <ItemGroup>
+    <TypeScriptCompile Remove="bin\Debug\dist\Remotion.Web.ClientScript.d.ts"/>
```

Comment:
Why do we need to remove this?

Reply:
We generate TypeScript files in the `bin` directory, which will be included in the project by the TypeScript infrastructure. It looks strange in the IDE and might affect the TypeScript build. As such, I added explicit removes here.